test/server/graphql/v1/mutation.test.js

```diff
@@ -96,6 +100,15 @@ describe('server/graphql/v1/mutation', () => {
     );
   });
 
+  beforeEach('create a project collective', async () => {
+    await models.Collective.create(
+      Object.assign(utils.data('project1'), {
+        CreatedByUserId: user1.id,
+        ParentCollectiveId: collective1.id,
+      }),
+    );
+  });
```

Comment:
We have a simpler way to create test entities nowadays that don't require adding new entries to `test/mocks/data.js`:

```suggestion
  beforeEach('create a project  under collective1', async () => {
    await fakeProject({ ParentCollectiveId: collective1.id });
  });
```